servers/rendering/renderer_compositor.h

```diff
@@ -59,7 +59,19 @@ class RendererCompositor {
 	struct BlitToScreen {
 		RID render_target;
 		Rect2i rect;
-		//lens distorted parameters for VR should go here
+
+		//stereo target
```

Comment:
put all the stereo stuff in a Stereo or XR substructure I guess

Reply:
Done